### PR TITLE
omitted reference of codewind-workspace and included updates of 0.6.0

### PR DESCRIPTION
**Pull request by j-c-berger**
Signed-off-by: Jacob Berger <jacob.berger@ibm.com>

@tetchel, please review changes when you can. 

As discussed in issue 846, https://github.com/eclipse/codewind/issues/846, I omitted the reference to `codewind-workspace` and then informed users how this workspace no longer needs to exist in 0.6.0.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Prerequisites
...
-3. Codewind requires the installation of additional Docker images to run. Choose **Install** when prompted to complete the installation. The installation may take a few minutes to complete. Codewind creates a folder called `codewind-workspace` within your home directory to contain your projects. On Windows, this is the `C:\codewind-workspace` directory. When the installation is complete, you can open the `codewind-workspace` folder or a project within the workspace as your VS Code workspace. The tools offer to open the workspace for you if it’s not open already.
+3. Codewind requires the installation of additional Docker images to run. Choose **Install** when prompted to complete the installation. The installation may take a few minutes to complete. Before 0.6.0, projects had to be created and saved in a fixed location on users' disk called, `codewind-workspace`. 0.6.0 allows you to have your project created and saved anywhere and also removes the need for your project to be in a container.
```
Comment (tetchel) [inline README.md]: I don't think we need to talk about the old behaviour here. READMEs generally just talk about what the product does right now. I would remove everything after "the installation may take a few minutes to complete".

Comment (j-c-berger) [reply README.md]: Cool, thanks, and will do!

**Comment by j-c-berger** [comment]
@tetchel went back and applied your feedback. See the new commit above. 

If it looks good to you, I can get @sishida to merge this PR. Thanks.

**Comment by sishida** [comment]
@j-c-berger: Hi, Jake, Tim is a committer, so he can approve and merge. :)

Tim, I'll leave this one to you. :)

**Comment by j-c-berger** [comment]
@sishida , yup, you're absolutely right! Duh!

**Pull request merged**